drivers/eaton-pdu-marlin-mib.c

```diff
@@ -282,8 +282,9 @@ long marlin_device_count_fun(const char *daisy_dev_list)
 # endif /* WITH_SNMP_LKP_FUN_DUMMY */
 
 static info_lkp_t marlin_device_count_info[] = {
-	{ 1, "dummy", NULL, marlin_device_count_fun },
-	{ 0, NULL, NULL, NULL }
+	/* TOTHINK: DMF branch used info_lkp_fun_s2l()? */
+	info_lkp_nuf_s2l(1, "dummy", marlin_device_count_fun),
+	info_lkp_sentinel
 };
```

Comment:
Here is the part about function wrappers (second or third pointer) I mentioned in another comment. The DMF/FTY branch has it different than this code, e.g.

* https://github.com/networkupstools/nut/blob/c4970c9b021e2875ae04208d72d889e072643c79/drivers/eaton-pdu-marlin-mib.c#L819-L848

Reply:
Note: the PR was later amended to at least use the pointer to function in non-DMF builds (conflate with the other `..._s2l()` method, though).